.github/workflows/client_node.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Node Client Generator and Create Pull Request
+on:
+  push:
+    branches:
+      - "master"
+jobs:
+  create_pr:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v2
+      with:
+        path: google-cloudevents
+    - name: Checkout client repo
+      uses: actions/checkout@v2
+      with:
+        repository: googleapis/google-cloudevents-nodejs
+        path: google-cloudevents-nodejs
+    - name: Setup Node for gen/postgen scripts
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - name: Run the generator
+      run: |
+        cd google-cloudevents-nodejs
+        npm i
+        npm run build
+        npm run gen
+    - name: Commit and create PR is needed
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        # If there is a non-zero diff after running generator
+        if [[ `git status --porcelain` ]]; then
+          git config --global user.email "yoshi-code-bot@google.com"
+          git config --global user.name "Google Bot"
+          git checkout -b "pr_bot-$GITHUB_RUN_ID"
+          git add -A
+          NOW=$(date +"%m-%d-%Y")
+          git commit --message "feat: run generator (${NOW})"
+          git push origin "pr_bot-$GITHUB_RUN_ID"
+          echo "Created commit: $(git rev-parse HEAD)"
+          gh pr create \
```

Comment:
nit: there's an action you can use, but this is just as clean

Reply:
Thanks. Using the CLI is easier for manual testing, so let's keep as is.